.github/workflows/tests.yaml

```diff
@@ -60,7 +60,7 @@ jobs:
 
     strategy:
       matrix:
-        python-version: [3.7, 3.8]
+        python-version: [3.7, 3.8, 3.9, '3.10']
```

Comment:
Does 3.10 need quotes? I don't like that it looks different.

Reply:
It does need quotes, otherwise it's parsed as 3.1. You can see this in the actions for a0a77d6. I can add quotes to all of them if you'd prefer

Reply:
Yes, let's do that - the single value with quotes is bugging me. Thanks.

Reply:
Fixed